examples/samples/SendersReceiversWithQueues.ts

```diff
@@ -0,0 +1,97 @@
+import { ReceiveMode, Namespace, generateUuid, QueueClient, SendableMessageInfo, OnMessage, ServiceBusMessage, OnError, MessagingError, delay, ReceiveHandler } from "../../lib";
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const str = process.env.SERVICEBUS_CONNECTION_STRING || "";
+const path = process.env.QUEUE_NAME || "";
+console.log("str: ", str);
+console.log("path: ", path);
+
+let ns: Namespace;
+
+async function sendMessages(queueClient: QueueClient): Promise<void> {
+  const data: any[] = [
```

Comment:
This can be `const data: { name: string; firstName: string }[]`

Also, shouldnt `name` here be `lastName`?